src/main/java/frc/robot/poseestimator/IPoseEstimator.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.poseestimator;
+
+import edu.wpi.first.math.geometry.Pose2d;
+
+
+public interface IPoseEstimator extends IVisionEstimator, IOdometryEstimator {
+
+	void resetPose(Pose2d newPose);
+
+	Pose2d getEstimatedPose();
+
+	Pose2d getEstimatedPoseAtTimeStamp(double timeStamp);
```

Comment:
Timestamp is usually one word 
```suggestion
	Pose2d getEstimatedPoseAtTimestamp(double timestamp);
```

Reply:
In fact you used this same spelling in other files; so this is more consistent too